py/test/test_cli_s3.py

```
@@ -0,0 +1,35 @@
+
+import pytest
+from ngff_zarr import cli
+
+
+@pytest.mark.parametrize(
+    ("uri", "expected_storage_options"),
+    [
+        (
+            "s3://aind-scratch-data/exaspim-ccfv4-template-dev/nifti-to-omezarr/"
+            "Registration_EXASPIM_delivery_ExaSPIM_template_10specimens_30um."
+            "ome.zarr",
```

Ruff (0.16.1)

[warning] Unparenthesized implicit string concatenation in collection

Did you forget a comma?

(ISC004)